charts/argo-cd/templates/argocd-application-controller/clusterrole.yaml

```diff
@@ -16,4 +19,5 @@ rules:
   - '*'
   verbs:
   - '*'
+{{- end }}
 {{- end }}
```

Comment:
can we pls add a new line at the end of the file, then  it will be more easy to take a look at the last line of file.

Reply:
yes just added it!